### ISSUE TITLE
How do I run benchmark

### ISSUE THREAD

**Issue by liushengahn** [eurecom-s3/symcc#95]
hello，
I am a beginner learning SymCC. I have prepared everything I need to run the benchmark(https://github.com/CyberGrandChallenge/samples) mentioned in the paper, but I failed. I don't know how to start.  I don't know how to run programs in CGC by SymCC. Could you help me ?
Thank you. Looking forward to your reply.

**Comment by liushengahn** [comment]
Please forgive me if the question is not suitable enough to ask.

**Comment by aurelf** [comment]
Hi, 
did you look here ? There is some information about the benchmarks:
https://www.s3.eurecom.fr/tools/symbolic_execution/symcc.html

**Comment by aurelf** [comment]
Also check docs and in particular [docs/Experiments.txt](https://github.com/eurecom-s3/symcc/blob/c4eacea9f888aa7676636e35c54b2c323a66b2f9/docs/Experiments.txt). 
Please reopen the bug if you don't find suitable information there.

**Comment by liushengahn** [comment]
hi, I built 32-bit versions of SymCC based on your tips, but I ran into some problems. I have built the 32-bit version of Z3, but I am not sure how to build the 32-bit version of LLVM. It looks simple in the document, but I don't know how to do it. https://github.com/eurecom-s3/symcc/blob/master/docs/32-bit.txt
![1642073105(1)](https://user-images.githubusercontent.com/46082729/149321789-6556ee78-561b-4450-8626-8cd883c8e481.jpg)
Thank you very much for your help.
